get-started/index.md

```diff
@@ -56,127 +56,156 @@ redirect_from:
-- Editing configuration files
-- Basic familiarity with the ideas of code dependencies and building
-- Machine resource usage terms, like CPU percentages, RAM use in bytes, etc.
-
-Finally, though we'll remind you again when you need these things, you can
-save yourself some distraction at that time by [signing up for a
-Docker ID](https://cloud.docker.com) and using it on your local machine
-by running the following command:
-
-```
-docker login
-```
-
-## A brief explanation of containers
+## Images and containers
 
 An **image** is a lightweight, stand-alone, executable package that includes
-everything needed to run a piece of software, including the code, a runtime,
-libraries, environment variables, and config files.
+everything needed to run a piece of software&#8212;the code, a runtime,
+libraries, environment variables, and configuration files. You can find
+thousands of images on [Docker Hub](https://hub.docker.com/explore/).
 
 A **container** is a runtime instance of an image&#8212;what the image becomes
-in memory when actually executed. It runs completely isolated from the host
-environment by default, only accessing host files and ports if configured to do
+in memory when executed. By default, a container runs completely isolated from
+the host environment, only accessing host files and ports if configured to do
 so.
 
-Containers run apps natively on the host machine's kernel. They have better
-performance characteristics than virtual machines that only get virtual access
-to host resources through a hypervisor. Containers can get native access, each
-one running in a discrete process, taking no more memory than any other
-executable.
+The use of Linux containers to build, deploy, and run applications is called
+[containerization](https://en.wikipedia.org/wiki/Operating-system-level_virtualization).
+Containers are not new but their use for easily deploying applications is.
 
 ## Containers vs. virtual machines
 
-Consider this diagram comparing virtual machines to containers:
+Each **virtual machine** (VM) runs a "guest" operating system (OS) with only
+virtual access to host resources through a hypervisor. VMs are resource
+intensive, and the resulting disk image and application state is an entanglement
+of OS settings, system-installed dependencies, OS security patches, and other
+easy-to-lose, hard-to-replicate ephemera. VMs provide an environment with far
+more resources than most applications need.
 
-### Virtual Machine diagram
+By contrast, a **container** runs applications natively and shares the kernel of
+the host machine with other containers, keeping it lightweight. Each container
+runs in a discrete process, taking no more memory than any other executable. The
+only information needed in a container image is the executable and its package
+dependencies, which never need to be installed on the host system. Because
+containers house their dependencies, a containerized app is portable and “runs
+anywhere.”
 
```

Comment:
Here we have an opportunity to talk about VMs and containers in combination (such as EC2). Want to add something about that? I am not sure we really need to be quite so negative about VMs here at all.

Reply:
Not yet but I"ll add it in a future version.